jax/experimental/jax2tf/jax2tf.py

```diff
@@ -904,13 +908,66 @@ def _reduce_window(jax_f, reducer, init_val, operand, window_dimensions,
                             window_dilations=window_dilation, padding=padding)
   out.set_shape(out_shape)
   return out
+
+def _get_max_identity(dtype):
```

Comment:
Let's rename this to `tf_dtype`. Perhaps the whole function can be written using only TF dtypes?

Reply:
Addressed in `eee7daa`